.travis.yml

```diff
@@ -18,7 +18,8 @@
 sudo: required
 
 group: deprecated-2017Q3
```

Comment:
can we drop this deprecated group?

Reply:
I think yes, since it is selecting a `trusty` variant and we are attempting to switch to `xenial`